src/rust/derive-dynamic-node/src/lib.rs

```diff
@@ -424,12 +424,12 @@ fn identity_prop_setter(field: &Field, property_name: &Ident) -> TS2 {
         _ => return quote!(),
     };
     quote!(
-        let mut self_strategy = mut_self.id_strategy[0].strategy.as_mut().unwrap();
+        let mut self_strategy = mut_self.id_strategy[0].strategy;
         match self_strategy {
-            rust_proto::graph_descriptions::id_strategy::Strategy::Session(
-                rust_proto::graph_descriptions::Session{ref mut #ident, ..}
+            rust_proto_new::graplinc::grapl::api::graph::v1beta1::Strategy::Session(
+                rust_proto_new::graplinc::grapl::api::graph::v1beta1::Session{ref mut #ident, ..}
             ) => {
-                * #ident = #property_name.as_inner();
+                * #ident = #property_name.as_inner().clone();
```

Comment:
this was an attempt to satisfy the borrow checker, it may not be the right thing to do here

Reply:
Looks good to me